pkg/controller/appmetricsservice/appmetricsservice_controller.go

```diff
@@ -2,13 +2,15 @@ package appmetricsservice
 
 import (
 	"context"
+	"time"
 
 	metricsv1alpha1 "github.com/aerogear/app-metrics-operator/pkg/apis/metrics/v1alpha1"
```

Comment:
I would recommend creating a specific controller for the BKP because of the main reasons as follows.  
- It goes implicitly against the design of the framework adopted. See https://github.com/operator-framework/operator-sdk/issues/1383#issuecomment-489604367
- It makes harder the usability since you need to say things as "the AppMetricsServiceCR" is an example, you need  to add it with or without a backup, just one can be applied and etc .. Instead of just have it as optional then when the user would like to have the BKP just apply its cr as it was done [here](https://github.com/aerogear/mobile-security-service-operator#configuring-the-backup-service) and [here](https://github.com/aerogear/mobile-security-service-operator/tree/master/pkg/controller/mobilesecurityservicebackup) for example.

Reply:
We only have one CRD, so that linked comment doesn't really apply to this situation. We won't be adding a separate controller & CRD for this, at least for now. In the future, when we rely on some external database operator, we'll inherit whatever backup ability that it provides.

Reply:
Following the comments inline.

- The framework design is for we have a controller for each CR/CRD. 
- Put all in the same place make really hard we keep it maintained and reduce the usability of it as described above. 

I would suggest you follow the steps in the MSS Oper to install and add the backup then try to do the same following the README of the UPS Oper to check it.